reference/constraints/Timezone.rst

```diff
@@ -0,0 +1,144 @@
+Timezone
+========
+
+Validates that a value is a valid timezone identifier (e.g. ``Europe/Paris``).
+
+==========  ======================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
```

Comment:
@symfony/team-symfony-docs do we use ordered options here and the same order for the explanations?

Reply:
We didn't reorder descriptions for existing docs in the past and only sorted the table of contents to keep the diff maintainable. But here it makes sense to sort from the beginning.

Reply:
Yes

Reply:
What do you mean?

Ordering options like:

1. Child class options first then inherited options?
2. The opposite order of `1.`?

Reply:
@xabbuh although you are right ... for the Constraints docs we recently sorted both the list of options and their descriptions. It was a hell to fix all the conflicts, but we did it anyways :)